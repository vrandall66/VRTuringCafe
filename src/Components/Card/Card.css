.Card {
  background-color: white;
  height: 200px;
  margin: 1%;
  width: 18%;
}