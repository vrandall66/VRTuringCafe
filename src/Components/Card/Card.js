import React from "react";
import "./Card.css";

const Card = ({ reservation, deleteReservation }) => {
  return (
    <div className="Card" key={reservation.id}>
      <h3>{reservation.name}</h3>
      <p>{reservation.date}</p>
      <p>Number of guests: {reservation.number}</p>
      <p>{reservation.time}</p>
      <button
        type="button"
        id="Card__btn--delete"
        onClick={() => deleteReservation(reservation.id)}
      >
        Cancel
      </button>
    </div>
  );
};

export default Card;
