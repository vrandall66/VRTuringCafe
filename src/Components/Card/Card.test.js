import React from "react";
import { shallow } from "enzyme";
import Card from "./Card";

describe("Card", () => {
  let wrapper;
  let reservation = {
    name: "",
    date: "",
    number: 0,
    time: "",
    id: 4
  };

  beforeEach(() => {
    wrapper = shallow(<Card reservation={reservation} key={reservation.id} />);
  });

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
