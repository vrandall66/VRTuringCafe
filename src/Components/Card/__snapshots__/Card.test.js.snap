// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<div
  className="Card"
  key="4"
>
  <h3 />
  <p />
  <p>
    Number of guests: 
    0
  </p>
  <p />
  <button
    type="button"
  >
    Cancel
  </button>
</div>
`;
